tests/test_configs.py

```
@@ -3,41 +3,73 @@
+                        self.assertIsInstance(
+                            model[field],
+                            expected_type,
+                            f"Field '{field}' should be {expected_type.__name__} in {config_path}",
+                        )
 
-    def test_vulkan_config_has_models(self):
-        self._load_config("Docker/vulkan/config-vulkan.json")
+                    self.assertEqual(model["model"], model["model"].strip())
+                    self.assertEqual(model["model_alias"], model["model_alias"].strip())
+                    self.assertNotIn(" ", model["model"], f"Unexpected space in model path for {config_path}")
+                    self.assertGreater(model["n_batch"], 0)
```

Ruff (0.15.7)

[warning] Use a regular `assert` instead of unittest-style `assertGreater`

Replace `assertGreater(...)` with `assert ...`

(PT009)